### DISCUSSION TITLE
Why do Web and iOS show two different themes when defaultTheme is set to "light" in Example App?

### DISCUSSION THREAD

**Discussion by reillyjodonnell** [tamagui/tamagui#535] [Q&A]
Hey everyone! I've just read through the docs and am now checking out the Example App you get with `yarn create tamagui`.

In the `index.tsx` of the `provider` folder we have this:

```
import config from '../tamagui.config'
import { NavigationProvider } from './navigation'
import { TamaguiProvider, TamaguiProviderProps } from '@my/ui'

export function Provider({ children, ...rest }: Omit<TamaguiProviderProps, 'config'>) {
  return (
    <TamaguiProvider config={config} disableInjectCSS defaultTheme="light" {...rest}>
      <NavigationProvider>{children}</NavigationProvider>
    </TamaguiProvider>
  )
}

``` 
By giving defaultTheme the value of "light" we are setting the top level theme to be "light". 

When I'm running the iOS simulator the screen is definitely light mode: 
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/65147216/212225536-e11c0ba6-047e-47ab-97ac-c6bad734318c.png">

But the web is in dark mode: 
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/65147216/212225653-aacc55df-f64c-4411-bf79-5cd2c8dc50dd.png">

Is there something that I'm missing / why this is the case?

**Comment by reillyjodonnell** [comment]
Ah! Just figured it out and am putting a PR together. In the Example App the `themeProvider` in `apps/next/pages/_app.tsx` is 
```
function ThemeProvider({ children }: { children: React.ReactNode }) {
  const [theme, setTheme] = useRootTheme()

  return (
    <NextThemeProvider
      onChangeTheme={(next) => {
        startTransition(() => {
          setTheme(next)
        })
      }}
    >
      <Provider disableRootThemeClass defaultTheme="light">
        {children}
      </Provider>
    </NextThemeProvider>
  )
}

``` 
but the `NextThemeProvider` should have the defaultTheme attribute added set to "light" if we wanted to be consistent - since NextThemeProvider is responsible for setting the class on the HTML tag.

Is there a way to keep the theme in the TamaguiProvider in sync with the NextThemeProvider?

**Comment by natew** [comment]
I think it’s because your system setting is dark so it grabs the preferred theme. TamaguiProvider takes a defaultTheme prop that can override.

**Reply by reillyjodonnell** [reply]
That's what I thought too but setting the themeProvider prop on the TamaguiProvider to "light" doesn't override the preferred theme in the example above. What am I missing here?

**Reply by esinanturan** [reply]
@natew  This issue still exists. TamaguiProvider has defaultTheme as light but still on the web it is using system theme. Can you take a look at this? ( I am using takeout boilerplate )
